src/views/SpikeAmplitudes/SpikeAmplitudesView.tsx

```diff
@@ -150,29 +150,29 @@ const SpikeAmplitudesViewChild: FunctionComponent<ChildProps> = ({data, timeseri
         return {yMin, yMax}
     }, [series])
 
-    const pixelTransform = use2dPanelDataToPixelMatrix(
-        pixelsPerSecond,
-        visibleTimeStartSeconds,
-        amplitudeRange.yMin,
-        amplitudeRange.yMax,
-        ampScaleFactor,
-        panelHeight,
-        true
-    )
-
+    const pixelTransform = use2dScalingMatrix({
+        totalPixelWidth: panelWidth,
+        totalPixelHeight: panelHeight,
+        dataXMin: visibleTimeStartSeconds,
+        dataXMax: visibleTimeEndSeconds,
+        dataYMin: amplitudeRange.yMin,
+        dataYMax: amplitudeRange.yMax,
+        yScaleFactor: ampScaleFactor,
```

Comment:
Note: the `yScaleFactor` logic in `pointProjection.ts` is copied from known-working code, but we will need to do more testing on the x-axis scaling. And I don't know how thoroughly I checked how these interplay with the code to maintain the aspect ratio.

Reply:
The y-axis scaling definitely will draw data points in the margins if that's where they are supposed to lie--because that's how it has always worked--but for x-axis scaling it might be smarter to just scale the underlying visible range instead.